File: Patterns/1. Square Patterns/08. Square Alphabet Pattern/sap.java
import java.util.*;
public class sap{
    public static void main(String [] args){
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter a Number : ");
        int n = sc.nextInt();

        for(int i=1; i<=n; i++){
            for(int j=1; j<=n; j++){
                System.out.print((char)('A'+j-1) + " ");
            }
            System.out.println();
        }

        sc.close();
    }
}

/*
Output

A B C D E 
A B C D E 
A B C D E 
A B C D E 
A B C D E   

*/

File: Patterns/1. Square Patterns/08. Square Alphabet Pattern/sap.py
n = int(input("Enter a Number : "))

for i in range(1,n+1):
    for j in range(1,n+1):
        print(chr(ord('A')+j-1),end=" ")
    print()


'''
Output:

A B C D E 
A B C D E 
A B C D E 
A B C D E 
A B C D E   

'''